content/en/account_management/saml/troubleshooting.md

```diff
@@ -0,0 +1,147 @@
+  - If a user is no longer enabled in a Datadog organization that has JIT provisioning enabled and they try to login again through SAML and the `There is no active account for error` occurs, re-enable the user in [User settings][3].
+
+## IdP metadata file errors
+
+If you are having trouble updating your IdP metadata file, verify that the metadata file you are trying to upload is valid.
+
+To validate your metadata file:
+
+1. Choose a SAML validation tool, such as the [SAML developer tool][4] by OneLogin.
+2. Paste your metadata into the XML field and select **Metadata** in the XSD (schema file) field.
+3. Click **Validate XML With the XSD Schema**.
+
+## Roles errors
+
+When mappings are enabled, users logging in with SAML to a Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider.
+
+Users who log in with SAML and do not have the values that map to a Datadog role are stripped of all roles and are not allowed to log in.
+
+{{< img src="account_management/saml/unknown_user_error.png" alt="No AuthNMappings for this user" style="width:80%;">}}
+
+If you have group mappings set and are not able to see your roles, your group mappings in the Datadog application may appear differently in your IdP. To verify:
+
+1. Retrieve your IdP's SAML assertion for your account. Use browser tooling, such as [extensions][5], to retrieve your SAML assertion. For example:
+
+  ```xml
+  <saml2:Attribute Name="member_of"
+                             NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+                             >
+                <saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                                      xsi:type="xs:string"
+                                      >name_of_your_group_goes_here</saml2:AttributeValue>
+  </saml2:Attribute>
+  ```
+
+2. Navigate to the Team page in the bottom left corner of Datadog.
+3. Select the [**Mappings**][6] tab.
+4. Compare the attributes provided by your IdP in your SAML assertion to the attributes set in the [**Mappings**][6] tab.
+
+  {{< img src="account_management/saml/saml_mappings_example.png" alt="SAML mappings in Datadog" style="width:80%;">}}
+
+5. Resolve any discrepancies in either the Datadog Mappings settings, or within your IdP settings. For example, if `memberof` is a set attribute in Datadog, and it's `member_Of` in your SAML assertion, resolve accordingly.
+
+Discrepancies may occur when there is no match or a mismatch between the attribute key and value. For example, if you see a key value pair of `memberOf` and `name_of_your_group_goes_here` in **Mappings**, you run into an issue because this pair is not included in the assertion sent over from your IdP.
```

Comment:
```suggestion
Discrepancies may occur when there is no match or a mismatch between the attribute key and value. For example, if you see a key value pair of `memberOf` and `name_of_your_group_goes_here` in **SAML Group Mappings**, you run into an issue because this pair is not included in the assertion sent over from your IdP.
```